apps/CEventService.API/obj/Debug/net8.0/CEventService.API.AssemblyInfo.cs

```diff
@@ -0,0 +1,22 @@
+//------------------------------------------------------------------------------
```

Comment:
remove this Debug/net8.0 folder from the PR

Reply:
Those files do not correspond to my files, it is as if I have pulled files from another branch.